canonical-websites/www.ubuntu.com
# Landing page for the 2nd AIML webinar

## Done

* [New landing page](http://www.ubuntu.com-canonical-websites-pr-4090.run.demo.haus/engage/deploy-ai-ml-model) based on the successful https://www.ubuntu.com/engage/ai-ml-ubuntu
* [Copy doc](https://docs.google.com/document/d/1BMqxhCOJfT43Mj2MFNgdM0X0c0rUYmBQBLeYFTCKfKY/edit#)

## QA

- Check out this feature branch
- Run the site using the command `./run serve`
- View the site locally in your web browser at: [http://0.0.0.0:8001/](http://0.0.0.0:8001/)
- Run through the following [QA steps](https://github.com/canonical-webteam/practices/blob/master/workflow/qa-steps.md)
- Ensure [the page](http://www.ubuntu.com-canonical-websites-pr-4090.run.demo.haus/engage/deploy-ai-ml-model) looks good and is matching the [copy doc](https://docs.google.com/document/d/1BMqxhCOJfT43Mj2MFNgdM0X0c0rUYmBQBLeYFTCKfKY/edit#)
